From ef75f8c98fda799049337d53a9a5279115ecf39d Mon Sep 17 00:00:00 2001
From: Hecheng Yu <kde-yyds@qq.com>
Date: Tue, 11 Feb 2025 09:33:28 +0800
Subject: [PATCH] remove RISING_RELEASE_TYPE

On some deives, they use LINEAGE_VERSION for mtdoops fingerprint. It's too long so that mtdoops fail to load. Remove this as a workaround
---
 config/version.mk | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/config/version.mk b/config/version.mk
index b6b847a..b810b93 100644
--- a/config/version.mk
+++ b/config/version.mk
@@ -8,7 +8,6 @@ PRODUCT_SOONG_NAMESPACES += \
 RISING_FLAVOR := VanillaIceCream
 RISING_VERSION := 6.2.1
 RISING_CODENAME := Boundless
-RISING_RELEASE_TYPE := STABLE
 RISING_CODE := $(RISING_VERSION)
 
 RISING_BUILD_DATE := $(shell date +%y%m%d%H)
@@ -42,10 +41,10 @@ else
 endif
 
 # Build version
-RISING_BUILD_VERSION := $(RISING_VERSION)-$(RISING_RELEASE_TYPE)-$(RISING_BUILD_DATE)-$(RISING_PACKAGE_TYPE)-$(RISING_BUILDTYPE)-$(CURRENT_DEVICE)
+RISING_BUILD_VERSION := $(RISING_VERSION)-$(RISING_BUILD_DATE)-$(RISING_PACKAGE_TYPE)-$(RISING_BUILDTYPE)-$(CURRENT_DEVICE)
 
 # Display version
-RISING_DISPLAY_VERSION := $(RISING_VERSION)-$(RISING_RELEASE_TYPE)-$(RISING_PACKAGE_TYPE)-$(RISING_BUILDTYPE)-$(CURRENT_DEVICE)
+RISING_DISPLAY_VERSION := $(RISING_VERSION)-$(RISING_PACKAGE_TYPE)-$(RISING_BUILDTYPE)-$(CURRENT_DEVICE)
 
 # RisingOS properties
 PRODUCT_SYSTEM_DEFAULT_PROPERTIES += \
-- 
2.48.1

